var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -171,20 +158,34 @@ def configure(self):
                 fh.write('\n'.join(makefile_inc))
 
     def install(self, spec, prefix):
-        self.configure()
-
         targets = ['scotch']
         if '+mpi' in self.spec:
             targets.append('ptscotch')
 
-        if '+esmumps' in self.spec:
-            targets.append('esmumps')
-            if '+mpi' in self.spec:
-                targets.append('ptesmumps')
+        if self.spec.version >= Version('6.0.0'):
+            if '+esmumps' in self.spec:
+                targets.append('esmumps')
+                if '+mpi' in self.spec:
+                    targets.append('ptesmumps')
 
         with working_dir('src'):
             for target in targets:
-                make(target, parallel=(target != 'ptesmumps'))
+                can_make_parallel = not (target == 'ptesmumps'
+                                         or (self.spec.version < Version('6.0.0')
+                                             and target == 'ptscotch'))
```

Comment:
Parallel builds for `ptscotch` worked fine for me with `@5.1.10b`.  Did you encounter any issues trying this build during your testing?